source/_components/sensor.google_calendar.markdown

```diff
@@ -0,0 +1,116 @@
+---
+layout: page
+title: "Google Calendar Event"
+description: "Instructions how to use Google Calendars in Home Assistant."
+date: 2015-05-08 17:15
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
+ha_category: Calendar
+ha_iot_class: "Cloud Polling"
+ha_release: unknown
+---
+
+
+This platform allows you to connect to your [Google Calendars](https://calendar.google.com) and generate binary sensors. The sensors created can trigger based on any event on the calendar or only for matching events. When you first setup this component it will generate a new configuration file *google_calendars.yaml* that will contain information about all of the calendars you can see.
+
+### {% linkable_title Prerequisites %}
+
+Generate a Client ID and Client Secret on [Google Developers Console](https://console.developers.google.com/start/api?id=calendar).
+
+1. To enable this sensor in your installation, add the following to your `configuration.yaml` file:
+1. When it gets to the point of asking _Which API are you using?_ just click cancel.
+1. Click on the tab 'OAuth consent screen'
+1. Set 'Product name shown to users' to anything you want. We suggest Home-Assistant.
+1. Save this page. You don't have to fill out anything else there.
+1. Click 'Create credentials' -> OAuth client ID
+1. Set the Application type to 'Other' and give this credential set a name then click Create
+1. Save the client ID and secret as you will need to put these in your configuration.yaml
```

Comment:
Suggest either changing this to *configuration file* or *configuration.yaml file*.